runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatOverlay.java

```diff
@@ -259,14 +259,14 @@ private String buildStatChangeString(StatChange c)
 		StringBuilder b = new StringBuilder();
 		b.append(ColorUtil.colorTag(Positivity.getColor(config, c.getPositivity())));
 
-		if (config.relative())
+		if (config.relative() && c.getPositivity() != Positivity.NO_CHANGE)
```

Comment:
I would just replace `config.relative()` with a variable like `boolean showRelative = config.relative() && c.getPositivity() != Positivity.NO_CHANGE`, and use that in it's place everywhere in this method. So that the setting appears as completely off when there is no change. This will also make it not add the unnecessary parentheses around the absolute value if there is no change and only that and the relative option are selected.

Reply:
I think I've gotten the gist of what you were suggesting. Sorry for the extreme delay; real life got in the way